buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -19,7 +19,7 @@ def build_and_test_win(){
 }
 
 def update_conda_recipes(){
-    sh "$WORKSPACE/buildconfig/Jenkins/Conda/update-conda-recipes.sh $GITHUB_ACCESS_TOKEN"
+    sh "$WORKSPACE/buildconfig/Jenkins/Conda/update-conda-recipes.sh $GITHUB_ACCESS_TOKEN mantid-builder"
```

Comment:
Could I suggest we also parameterize the user name, i.e. have `$GITHUB_USER_NAME` instead of hardcoding `mantid-builder`? It just makes it much easier to update configs in the future. I can add the job parameter.

Reply:
Please add the job parameter

Reply:
It should be there now.